modules/group/vars.tf

```diff
@@ -12,3 +12,20 @@ variable email {
   type        = string
   description = "Email address of GSuite Group"
 }
+
+variable "members" {
```

Comment:
remove the quotes around the name (or is it something specific in v12 for type = object?)